acmwebsite/templates/master.xhtml

```diff
@@ -9,14 +9,15 @@
     <link id='bs-css' rel="stylesheet" type="text/css" media="screen" href="${tg.url('/css/bootstrap.dark.min.css') if tg.session.get('theme', None) == 'dark' else tg.url('/css/bootstrap.light.min.css')}" />
     <link rel="stylesheet" type="text/css" media="screen" href="${tg.url('/css/style.css')}" />
     <link rel="stylesheet" type="text/css" media="screen" href="${tg.url('/css/font-awesome.min.css')}" />
+    <link rel="stylesheet" type="text/css" media="screen" href="${tg.url(tg.config.get('custom_assets.css'))}" py:strip="not tg.config.get('custom_assets.css')"/>
   </head>
 
   <body>
     <!-- Navbar -->
     <div class="container">
       <div class="header-logo">
         <a href="${tg.url('/')}">
-          <img src="${tg.url('/img/full.svg')}" alt="Mines ACM" class="acm-logo" />
+            <img src="${tg.url(tg.config.get('custom_assets.logo', 'logo.jpg'))}" alt="${tg.config.get('site.name')}" class="club-logo" />
```

Comment:
See below comments, this will need changed.

Reply:
Just use the CSS class "logo". Also make sure to change the css class if you adjust this. BOTH sides need adjusted.